### PR TITLE
ft(feature):create a political party

### PR DESCRIPTION
**Pull request by Hoxtygen**
#### What does this PR do?
Add a new route to the endpoint.This new route allows the creation of
new political office.

#### Description of Task to be completed?
Have the following endpoint working
`POST /api/v1/parties/`

#### How should this be manually tested?
  clone the repo and cd into it then on the terminal do `npm run dev`
Using postman, test the endpoint with the following header information
- key: Content-Type value: application/json

#### What are the relevant pivotal tracker stories?
[Delivers [#163393386](https://www.pivotaltracker.com/story/show/163393386)]

#### Screenshots (if appropriate)

createparty

### CODE REVIEW

File: server/controllers/partiesController.js [hunk 1]
```diff
@@ -43,6 +43,31 @@ class partyController {
...
+
+    const newParty = {
+      id: Parties.length + 1,
+      name,
+      Acronym,
+      hqAddress,
+      logoUrl,
+    };
+    Parties.push(newParty);
+    console.log();
```
Comment (Orlayhemmy) [inline server/controllers/partiesController.js]: Remove the log

File: server/controllers/partiesController.js [hunk 2]
```diff
@@ -43,6 +43,31 @@ class partyController {
       error: 'Party not found',
     });
   }
+
+  static addNew(req, res) {
```
Comment (Orlayhemmy) [inline server/controllers/partiesController.js]: Add new what? Give a better variable naming

**Review by Orlayhemmy — CHANGES_REQUESTED** [review]

File: server/controllers/partiesController.js (hunk 1 above)
Comment (Hoxtygen) [reply server/controllers/partiesController.js]: Okay. That one got away.

File: server/controllers/partiesController.js (hunk 2 above)
Comment (Hoxtygen) [reply server/controllers/partiesController.js]: Copy that.

**Pull request merged**